index.js

```diff
@@ -30,14 +29,40 @@ converters.Program = (path, context) => {
 
       let params = path.get('superTypeParameters').get('params');
       let props = params[0];
+      let defaultProps = [];
+
+      path.get('body').node.body.find(a => {
+        if (a.type === 'ClassProperty' && a.key.name === 'defaultProps') {
+          defaultProps = a.value.properties
+        }
+      });
+
+      const classProperties = convert(props, Object.assign({}, context, { mode: 'type' }))
 
-      result.classes.push(convert(props, context));
+      defaultProps.forEach(property => {
+        const prop = classProperties.props.find(p => p.key === property.key.name)
+        const value = convert(property,  Object.assign({}, context, { mode: 'value' }));
+        if (value) prop.default = value;
+      });
+
+      result.classes.push(classProperties);
     },
   });
-
   return result;
 };
 
+converters.ObjectProperty = (path, context) => {
+  const key = convert(path.key, context)
+  const value = generate(path.value).code;
+  return { key, value, kind: path.value.type }
+}
+
+converters.ObjectMethod = (path, context) => {
+  const key = convert(path.key, context)
+  const value = generate(path.body).code;
+  return { key, value, kind: path.body.type }
```

Comment:
Can you manually specify the different `kind`'s? I don't want to reuse the Babel ones in some places and not others